packages/http-client/src/http-client.spec.ts

```diff
@@ -678,6 +688,33 @@ sap-client:001`);
         })
       );
     });
+
+    xit('test axios proxy redirect', () => {
```

Comment:
I guess you might have created a follow up for this.
Could you please add some comments about the purpose of this disabled test and maybe link to the ticket?

Reply:
Added a comment to the test. Have created a [ticket](https://github.com/SAP/cloud-sdk-backlog/issues/560) and linked this test